modules/oadp-1-4-1-release-notes.adoc

```diff
@@ -0,0 +1,128 @@
+link:https://issues.redhat.com/browse/OADP-4076[OADP-4076]
+
+
+[id="resolved-issues-1-4-1_{context}"]
+== Resolved issues
+
+.PodVolumeBackup CR now displays correct message
+
+Previously, the `PodVolumeBackup` CR had an incorrect message, `get a podvolumebackup with status "InProgress" during the server starting, mark it as "Failed"`.
+With this update, the message is now updated as follows:
+[source,terminal]
+----
+found a podvolumebackup with status "InProgress" during the server starting,
+mark it as "Failed".
+----
+link:https://issues.redhat.com/browse/OADP-4224[OADP-4224]
+
+.Overriding imagePullPolicy is now possible with DPA  
+
+Previously, OADP used to set the `imagePullPolicy` parameter to `Always` for all images.
+With this update, OADP now checks if each image contains `sha256` or `sha512` digest then it sets `imagePullPolicy` to `IfNotPresent`; otherwise sets it to `Always`. You can now override this policy by using the new `spec.containerImagePullPolicy` DPA field.
+link:https://issues.redhat.com/browse/OADP-4172[OADP-4172]
+
+.OADP Velero can now retry updating the restore status if initial update fails
+
+Previously, OADP Velero failed to update the restored CR status and the status was stuck at `InProgress` indefinitely. As a consequence, it affected other components' behavior that relied on the backup and restore CR status to determine the completion. 
+With this update, the restore CR status would not be `InProgress` indefinitely and should eventually be `Complete` or `Failed`.
+link:https://issues.redhat.com/browse/OADP-3227[OADP-3227]
+
+.Restoring BuildConfig Build from a different cluster is successful without any errors
+ 
+Previously, on performing a restore of the BuildConfig Build resource from a different cluster, the application resulted in errors on TLS verification to the internal image registry. It resulted in the `failed to verify certificate: x509: certificate signed by unknown authority` error.
+With this update, the application restores successfully without any errors.
+link:https://issues.redhat.com/browse/OADP-4692[OADP-4692]
+
+.Restoring an empty PVC is successful 
+
+Previously, downloading data failed while restoring an empty persistent volume claim (PVC). It failed with the following error:
+[source,terminal]
+----
+data path restore failed: Failed to run kopia restore: Unable to load
+    snapshot : snapshot not found
+----
+With this update, an empty PVC is restored correctly and data download does not fail.
+link:https://issues.redhat.com/browse/OADP-3106[OADP-3106]
+
+.There is no Velero memory leak in CSI and DataMover plugins
+
+Previously, Velero memory leak was caused while using the CSI and DataMover plugins. When the backup ended, the Velero plugin instance was not deleted and it caused the memory to grow till OOM (Velero pod). With this update, there is no Velero memory leak.
```

Comment:
```suggestion
Previously, Velero memory leak was caused while using the CSI and DataMover plugins. When the backup ended, the Velero plugin instance was not deleted and it caused the memory to grow till out-of-memory (OOM) (Velero pod). With this update, there is no Velero memory leak.
```

Reply:
based on the existing occurrence in OCP, I've used Out of Memory (OOM).